setup.py

```diff
@@ -370,6 +370,7 @@ def run_setup(with_extensions):
         maintainer_email='alimanfoo@googlemail.com',
         url='https://github.com/zarr-developers/numcodecs',
         license='MIT',
+        zip_safe=False,
```

Comment:
This might not be needed. Just trying something 🍀

That said, we probably already should have had this given we have shared libraries

Or at least shared libraries misbehaving was what had usually prompted this before. Maybe it is [not as relevant anymore]( https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html )